src/presentation/public/manifest.json

```diff
@@ -0,0 +1,62 @@
+{
+    "name": "privacy.sexy",
+    "short_name": "privacy.sexy",
+    "description": "Enforce privacy & security best-practices on Windows, macOS and Linux, because privacy is sexy.",
+    "version": "0.13.5",
```

Comment:
This version should be bumped automatically. It would be better use some kind of logic in the vite build system to read the latest version and output it here.